plugins/optimization-detective/hooks.php

```diff
@@ -15,3 +15,4 @@
 OD_URL_Metrics_Post_Type::add_hooks();
 add_action( 'wp', 'od_maybe_add_template_output_buffer_filter' );
 add_action( 'wp_head', 'od_render_generator_meta_tag' );
+add_filter( 'od_url_metric_schema_root_additional_properties', 'od_add_optional_url_metric_schema_root_properties' );
```

Comment:
Per above, since this ETag is a core property I think it doesn't need to use a filter.